### PR TITLE
Add Protocol Mocks section to Deployed Contracts page

### PR DESCRIPTION
**Pull request by Psirex**

### CODE REVIEW

File: docs/deployed-contracts.md [hunk 1]
```diff
@@ -97,3 +97,12 @@
 - Curve Liquidity Farming:
 
   - Pool Contract: [`0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E`](https://goerli.etherscan.io/address/0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E)
+
+## Protocol Mocks
+
+### Rinkeby
+
+- Lido: [`0xF4242f9d78DB7218Ad72Ee3aE14469DBDE8731eD`](https://rinkeby.etherscan.io/address/0xF4242f9d78DB7218Ad72Ee3aE14469DBDE8731eD)
```
Comment (kadmil) [inline docs/deployed-contracts.md:105]: add `StETH`, please

File: docs/deployed-contracts.md [hunk 2]
```diff
@@ -97,3 +97,12 @@
 - Curve Liquidity Farming:
 
   - Pool Contract: [`0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E`](https://goerli.etherscan.io/address/0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E)
+
+## Protocol Mocks
+
+### Rinkeby
```
Comment (kadmil) [inline docs/deployed-contracts.md:103]: Do we have Oracle here?

File: docs/deployed-contracts.md (hunk 1 above)
Comment (Psirex) [reply docs/deployed-contracts.md]: I've updated the naming for this contract and others, to make it more consistent with the names of contracts in other sections.

File: docs/deployed-contracts.md (hunk 2 above)
Comment (Psirex) [reply docs/deployed-contracts.md:103]: We don't have a standalone contract for oracle in protocol mock, EOA address can simulate its reports.

**Review by kadmil — APPROVED** [review]
LGTM

**Pull request merged**